tests/test.py

```diff
@@ -2299,6 +2299,65 @@ def testReadAcrastructInAcrastruct(self):
         self.assertEqual(row['empty'], b'')
 
 
+class KMSAWSType:
+    def setUp(self):
+        if not TEST_WITH_AWS_KMS:
+            self.skipTest("test with AWS KMS ACRA_MASTER_KEY loader")
+
+        configuration = {
+            'access_key_id': 'access_key_id',
+            'secret_access_key': 'secret_key_id',
+            'region': os.environ.get('KMS_REGION', 'eu-west-2'),
+            'endpoint':  os.environ.get('AWS_KMS_ADDRESS', 'http://localhost:8080')
+        }
+        self.config_file = tempfile.NamedTemporaryFile('w+', encoding='utf-8')
+        json.dump(configuration, self.config_file)
+        self.config_file.flush()
+        super().setUp()
+
+    def get_kms_type(self):
+        return 'aws'
+
+    def get_kms_configuration_path(self):
+        return self.config_file.name
+
+
+class KMSEncryptorMixing:
+    ZONES = []
+
+    def setUp(self):
+        self.keystore = tempfile.TemporaryDirectory()
+        self.keys_dir = os.path.join(self.keystore.name, '.acrakeys')
+
+        extra_args = {
+            'kms_keystore_encryptor' : 'true',
+            'kms_type': self.get_kms_type(),
+            'kms_credentials_path': self.get_kms_configuration_path(),
+        }
+        assert create_client_keypair_from_certificate(TEST_TLS_CLIENT_CERT, keys_dir=self.keys_dir, extra_kwargs=extra_args) == 0
```

Comment:
hm, will it conflict with already existing keys in keystore with different arguments for keymaker (e.g. without any kms)?